### PR TITLE
TiUP:  add three docs for cluster start, stop, and more info

### PR DESCRIPTION
**Pull request by qiancai**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.0 (TiDB 5.0 versions)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/4868
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tiup/tiup-component-cluster-start.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+The `tiup cluster start` command is used to start all services or some services of the specified cluster.
+
+## Syntax
+
+```sh
+tiup cluster start <cluster-name> [flags]
+```
+
+``<cluster-name>` is the name of the cluster to operate on. If you forget the cluster name, check it in [tiup cluster list](/tiup/tiup-component-cluster-list.md).
```
Comment (qiancai) [inline tiup/tiup-component-cluster-start.md:15]:
```suggestion
`<cluster-name>` is the name of the cluster to operate on. If you forget the cluster name, check it in [tiup cluster list](/tiup/tiup-component-cluster-list.md).
```

File: tiup/tiup-component-cluster.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+- Skips the secondary confirmation of all risky operations. Using this option is not recommended, unless you are using a script to call TiUP.
+- This option is disabled by default with the `false` value. To enable this option, add this option to the command, and either pass the `true` value or do not pass any value.
+
+### -v, --version
+
+- Prints the current version of TiUP Cluster.
+- Data type: `BOOLEAN`
+- This option is disabled by default with the `false` value. To enable this option, add this option to the command, and either pass the `true` value or do not pass any value.
+
+### -h, --help (boolean, default false)
```
Comment (qiancai) [inline tiup/tiup-component-cluster.md:54]:
```suggestion
### -h, --help
```

**Review by Joyinqin — COMMENTED** [review]
rest lgtm

**Comment by Joyinqin** [comment]
@qiancai please address the ci failure~

File: tiup/tiup-component-cluster.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+
+### --ssh
+
+- Specifies the SSH client to connect to the remote end (the machine where the TiDB service is deployed) for the command execution.
+- Data type: `STRING`
+- Supported values:
+
+  - `builtin`: uses the easyssh client built in tiup-cluster as the SSH client.
+  - `system`: uses the default SSH client of the current operating system.
+  - `none`: The SSH client is not used. The deployment is only for the current machine.
```
Comment (TomShawn) [inline tiup/tiup-component-cluster.md:27]:
```suggestion
    - `builtin`: uses the easyssh client built in tiup-cluster as the SSH client.
    - `system`: uses the default SSH client of the current operating system.
    - `none`: The SSH client is not used. The deployment is only for the current machine.
```

File: tiup/tiup-component-cluster-stop.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+tiup cluster stop <cluster-name> [flags]
+```
+
+`<cluster-name>` is the name of the cluster to operate on. If you forget the cluster name, check it with the [cluster list](/tiup/tiup-component-cluster-list.md) command.
+
+## Options
+
+### -N, --node
+
+- Specifies the nodes to be restarted. The value of this option is a comma-separated list of node IDs. You can get the node IDs from the first column of the [cluster status table](/tiup/tiup-component-cluster-display.md) returned by the `tiup cluster display` command.
```
Comment (lucklove) [inline tiup/tiup-component-cluster-stop.md:25]:
```suggestion
- Specifies the nodes to be stoped. The value of this option is a comma-separated list of node IDs. You can get the node IDs from the first column of the [cluster status table](/tiup/tiup-component-cluster-display.md) returned by the `tiup cluster display` command.
```

File: tiup/tiup-component-cluster-stop.md [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+- Data type: `STRING`
+- Default: []. If this option is not specified in the command, the command stops all the nodes.
+
+> **Note:**
+> 
+> If the `-R, --role` option is specified at the same time, only the service nodes that match both the specifications of `-N, --node` and `-R, --role` are restarted.
+
+### -R, --role
+
+- Specifies the roles of nodes to be restarted. The value of this option is a comma-separated list of the roles of the nodes. You can get the roles of the nodes from the second column of the [cluster status table](/tiup/tiup-component-cluster-display.md) returned by the `tiup cluster display` command.
```
Comment (lucklove) [inline tiup/tiup-component-cluster-stop.md:35]:
```suggestion
- Specifies the roles of nodes to be stoped. The value of this option is a comma-separated list of the roles of the nodes. You can get the roles of the nodes from the second column of the [cluster status table](/tiup/tiup-component-cluster-display.md) returned by the `tiup cluster display` command.
```

File: tiup/tiup-component-cluster-start.md [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+- Data type: `STRING`
+- Default: `[]`. If this option is not specified in the command, all nodes are started.
+
+> **Note:**
+>
+> If the `-R, --role` option is specified at the same time, only the service nodes that match both the specifications of `-N, --node` and `-R, --role` are restarted.
+
+### -R, --role
+
+- Specified the roles of nodes to be restarted. The value of this option is a comma-separated list of the roles of the nodes. You can get the roles of the nodes from the second column of the [cluster status table](/tiup/tiup-component-cluster-display.md) returned by the `tiup cluster display` command.
```
Comment (lucklove) [inline tiup/tiup-component-cluster-start.md:31]:
```suggestion
- Specified the roles of nodes to be started. The value of this option is a comma-separated list of the roles of the nodes. You can get the roles of the nodes from the second column of the [cluster status table](/tiup/tiup-component-cluster-display.md) returned by the `tiup cluster display` command.
```

File: tiup/tiup-component-cluster-start.md [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+tiup cluster start <cluster-name> [flags]
+```
+
+`<cluster-name>` is the name of the cluster to operate on. If you forget the cluster name, check it using the [cluster list](/tiup/tiup-component-cluster-list.md) command.
+
+## Options
+
+### -N, --node
+
+- Specifies the nodes to be restarted. The value of this option is a comma-separated list of node IDs. You can get the node IDs from the first column of the [cluster status table](/tiup/tiup-component-cluster-display.md) returned by the `tiup cluster display` command.
```
Comment (lucklove) [inline tiup/tiup-component-cluster-start.md:21]:
```suggestion
- Specifies the nodes to be started. The value of this option is a comma-separated list of node IDs. You can get the node IDs from the first column of the [cluster status table](/tiup/tiup-component-cluster-display.md) returned by the `tiup cluster display` command.
```

**Pull request merged**